CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Enhance contribution guidelines: Refactored shell scripts, added additional steps.
```

Comment:
Please add the `#91` back, like it was before, at the end of the line

Reply:
Done